frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
```

Comment:
the rules for this one are now quite lax - there's some interesting stuff in there to enforce consistent type imports (`import type {Foo, bar} from 'package';` vs. `import {type Foo, Bar} from 'package';`) - now this project is using a mix of both.